sdlib/d/gc/tcache.d

```diff
@@ -54,6 +64,39 @@ public:
 		pd.arena.free(emap, pd, ptr);
 	}
 
+	// Determine whether given alloc is appendable
+	bool is_appendable(void* ptr) {
```

Comment:
This is not used anywhere, and is just a helper around another function, that doesn't seems worth including.